resources/function_help/json/apply_dash_pattern

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "apply_dash_pattern",
+  "type": "function",
+  "groups": ["GeometryGroup"],
+  "description": "Applies a dash pattern to a geometry, returning a MultiLineString geometry which is the input geometry stroked along each line/ring with the specified pattern.",
+  "arguments": [
+    {"arg":"geometry","description":"a geometry"},
```

Comment:
A {single,multi}line geometry only, or does it accept polygons too? I'd specify here.